### ISSUE TITLE
How can I change the storage locationfor tailscale android app

### ISSUE THREAD

**Issue by byteshiva** [tailscale/tailscale#5822]
### What is the issue?

I would like to change the TailScale android app primary storage  location to sdcard which has more than capacity.

Do i need to rebuild it or are there any ways to accomplish it

### Steps to reproduce

go to tailscale android app and try change the primary storage  location/

### OS

Android

### OS version

Android

### Tailscale version

1.30.2

**Comment by bradfitz** [comment]
What phone do you have?

**Comment by byteshiva** [comment]
> What phone do you have?

Android - **Redmi** 9 (used for  testing )

**Comment by DentonGentry** [comment]
The primary concern is storage of files from Taildrop?

There isn't currently a setting in the app to control storage location, right now you'd need to build the app from source.

**Comment by byteshiva** [comment]
> isn't currently a setting in the app to control

Yes you are right, I was trying to `tailscale cp` files(larger than 1 GB) from desktop(n/w 1) to phone (n/w 2) and due to space constraint the copy was very slow. So I thought to change storage location in Tailscale.
